src/components/common/container/PreviewContainer.tsx

```diff
@@ -69,9 +68,27 @@ const FooterContainer = styled.div`
   font-size: 15px;
   color: ${Color.GREY};
   gap: 10px;
+  cursor: pointer;
+  &:hover {
+    color: ${Color.KIO_ORANGE};
+  }
+`;
+
+const PreviewContent = styled.iframe`
+  width: 100%;
+  height: 100%;
+  border: none;
 `;
 
-function PreviewContainer({ children, width = 360, height = 700 }: PreviewContainerProps) {
+function PreviewContainer({ width = 360, height = 700 }: PreviewContainerProps) {
+  const baseUrl = `${location.protocol}//${location.host}`;
```

Comment:
```suggestion
  const baseUrl = `${location.origin}`;
```
이렇게도 쓸 수 있을 것 같아요!
[location 속성](https://developer.mozilla.org/en-US/docs/Web/API/Location)